spec/lib/protobuf/field/field_hash_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+RSpec.describe Protobuf::Field::FieldHash do
+
+  class FieldHashBasicMessage < ::Protobuf::Message
```

Comment:
You might not have to clutter the namespace if you do something like this, and `stub_const('FieldHashBasicMessage', :field_hash_basic_message)`:

https://github.com/ruby-protobuf/protobuf/blob/963322cb9053e1a0950f0985320bd34c0f075543/spec/lib/protobuf/field/message_field_spec.rb#L4